index.html

```diff
@@ -1,386 +1,367 @@
+                            alt="jslovers homepage"
+                            width="100"
                         />
-                        <h4 class="mt-2 underline">Microsoft</h4>
 
-                   
+                        <h4 class="mt-2 underline">Microsoft</h4>
                     </div>
                     <div
                         class="w-32 flex flex-col justify-center items-center rounded-xl font-work-sans p-2 cursor-pointer"
                     >
                         <img
-                            src="./resources/assets/expedia.png"
-                            class="w-24 h-24 rounded object-contain"
-                            alt=""
+                            src="./resources/assets/jsloversLogo.png"
+                            alt="jslovers homepage"
+                            width="100"
                         />
+
                         <h4 class="mt-2 underline">Expedia</h4>
-                     
                     </div>
                     <div
                         class="w-32 flex flex-col justify-center items-center rounded-xl font-work-sans p-2 cursor-pointer"
                     >
                         <img
-                            src="./resources/assets/google.png"
-                            class="w-24 h-24 rounded object-contain"
-                            alt=""
+                            src="./resources/assets/jsloversLogo.png"
+                            alt="jslovers homepage"
+                            width="100"
                         />
-                        <h4 class="mt-2 underline">Google</h4>
-                     </div>
 
+                        <h4 class="mt-2 underline">Google</h4>
+                    </div>
                     <div
                         class="w-32 flex flex-col justify-center items-center rounded-xl font-work-sans p-2 cursor-pointer"
                     >
                         <img
-                            src="./resources/assets/ticketmaster.png"
-                            class="w-24 h-24 rounded object-contain"
-                            alt=""
+                            src="./resources/assets/jsloversLogo.png"
+                            alt="jslovers homepage"
+                            width="100"
                         />
-                    <h4 class="mt-2 underline">Ticket master</h4>
-                
+
+                        <h4 class="mt-2 underline">Ticketmaster</h4>
                     </div>
                     <div
                         class="w-32 flex flex-col justify-center items-center rounded-xl font-work-sans p-2 cursor-pointer"
                     >
                         <img
-                            src="./resources/assets/Nagarro.png"
-                            class="w-24 h-24 rounded object-contain"
-                            alt=""
+                            src="./resources/assets/jsloversLogo.png"
```

Comment:
You need to take update as these are the old changes which we have already updated